docs/appendix/upgrade-notes/2.x-to-3.0.rst

```diff
@@ -203,6 +206,7 @@ Deprecated APIs
  * ``elgg_list_entities_from_relationship``: Use ``elgg_list_entities()``
  * ``elgg_list_entities_from_private_settings``: Use ``elgg_list_entities()``
  * ``elgg_list_entities_from_access_id``: Use ``elgg_list_entities()``
+ * ``elgg_batch_delete_callback``
```

Comment:
why remove this function? It can be helpful sometimes

Reply:
I think it creates a anti-pattern. It makes it harder to read the code when used with ElggBatch. I'd rather we write explicit code with object instances that are type hinted.